src/main/kotlin/nl/info/zac/policy/PolicyService.kt

```diff
@@ -67,11 +66,16 @@ class PolicyService @Inject constructor(
      * Read 'overige' permissions.
      * Note that for PABC-based authorization, the 'zaaktype' parameter is required, but for legacy
      * ZAC-only authorization it is ignored.
-     * Until the old legacy authorization has been removed it is therefore nullable.
+     * Until the old legacy authorization has been removed, it is therefore nullable.
      */
-    fun readOverigeRechten(zaaktype: String? = null): OverigeRechten =
+    fun readOverigeRechten(zaaktype: String? = null) =
```

Comment:
It's a shame we didn't solve this with an abstract base class that would delegate implication to specific pabc and legacy implications....  this way the logic gets muddled and harder to follow

Reply:
yes agreed

Reply:
but more importantly: how come our integration tests did not catch this?